### PR TITLE
exclusions integration work

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-4306

### Change description ###

Exclusions integration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by kuda1992 — APPROVED** [review]
Looks good to me

File: api/roleAccess/index.ts [hunk 1]
```diff
@@ -6,9 +6,57 @@ import { getConfigValue } from '../configuration';
...
 
+export async function getRolesByCaseId(req: EnhancedRequest, res: Response, next: NextFunction): Promise<Response> {
+  const requestPayload = getLegalAndJudicialRequestPayload(req.body.caseId, req.body.jurisdiction, req.body.caseType);
+  const basePath = getConfigValue(SERVICES_ROLE_ASSIGNMENT_API_PATH);
+  const fullPath = `${basePath}/am/role-assignments/query`;
+  const headers = setHeaders(req, release2ContentType);
+  try {
+    console.log(JSON.stringify(requestPayload))
+    const response: AxiosResponse = await http.post(fullPath, requestPayload, {headers});
+    console.log(response.data);
```
Comment (connorpgpmcelroy) [inline api/roleAccess/index.ts:27]: Minor comment - Are the console.logs necessary?

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM but left minor comment

File: api/roleAccess/index.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/roleAccess/index.ts]: Removed it

**Pull request merged**